doc/man-sections/client-options.rst

```diff
@@ -68,7 +68,13 @@ configuration.
       auth-user-pass up
 
   If ``up`` is present, it must be a file containing username/password on 2
-  lines. If the password line is missing, OpenVPN will prompt for one.
+  lines. If the password line is missing, OpenVPN will prompt for one on
+  stdin.
+
+  *NOTE*: If you use a graphical OpenVPN client such as OpenVPN GUI on Windows,
+  you should either define both the username and password in the file, or rely
```

Comment:
The *NOTE*: should be on a single line, and the text should be indented one more level to be correctly rendered.  

The current patch gives:
```
              If up is present, it must be a file containing username/password
              on 2 lines. If the password line is missing, OpenVPN will prompt
              for one on stdin.

              NOTE: If you use a graphical OpenVPN client such as OpenVPN  GUI
              on Windows, you should either define both the username and pass‐
              word in the file, or rely on the GUI  caching  the  credentials;
              defining just the username in the file will not work.

              If  up  is  omitted, username/password will be prompted from the
              console.
```

With suggested changes:
```
              If up is present, it must be a file containing username/password
              on 2 lines. If the password line is missing, OpenVPN will prompt
              for one on stdin.

              NOTE:  If you use a graphical OpenVPN client such as OpenVPN GUI
                     on  Windows,  you  should either define both the username
                     and password in the file, or rely on the GUI caching  the
                     credentials; defining just the username in the file

              If  up  is  omitted, username/password will be prompted from the
              console.
```

Reply:
Right, there is some inconsistency there.  I think we should move all of these NOTE/WARN/INFO (whatever tags we use) to an indented approach, to hightlight it better.

This also gives us better flexibility when we start adding some CSS styling to the HTML rendering, where the indented variant rendering gives much better abilities the formatting and styling.  Essentially it changes from a more plain `<p>...</p>`  for "everything" to a `<dl><dt>$HEADER</dt><dd>$CONTENT</dd></dl>` style.

Reply:
should we keep this patch as is and then reformat all the NOTEs with a follow up?

Reply:
@mattock would you mind rebasing this patch on master and sending it to the mailing list? Once that's done, we can close this PR. Thanks!